docs/concepts/policy/resource-quotas.md

```diff
@@ -218,6 +221,30 @@ services                0       10
 services.loadbalancers  0       2
 ```
 
+Kubectl also supports object count quota for all namespaced resources
+using the syntax `count/<resource>.<group>`:
+
+```shell
+$ kubectl create namespace myspace
```

Comment:
please remove the `$` characters prefixing the commands (per style guide: https://kubernetes.io/docs/home/contribute/style-guide/#dont-include-the-command-prompt)